### PR TITLE
integrals: Fixed pyflakes errors in rubi.py and utility_function.py i…

### LINKED ISSUE #13283: Fix all pyflakes errors in the rubi module
The rubi module has a lot of code. If you run pyflakes on it, it reports a *lot* of errors, including undefined names. 

pyflakes should be giving no errors anywhere in the rubi module. That way we can be sure there aren't basic logical errors.

### PR DESCRIPTION
**Pull request by alirizwi**
Fixes https://github.com/sympy/sympy/issues/13283

### CODE REVIEW

File: sympy/integrals/rubi/utility_function.py [hunk 1]
```diff
@@ -4540,7 +4539,7 @@ def KnownTrigIntegrandQ(lst, u, x):
     A_ = Wild('A', exclude=[x])
     B_ = Wild('B', exclude=[x])
     C_ = Wild('C', exclude=[x])
-    F_ = Wild('F')
+    #F_ = Wild('F')
```
Comment (parsoyaarihant) [inline sympy/integrals/rubi/utility_function.py]: you can remove this line if this variable is not in use

File: sympy/integrals/rubi/utility_function.py [hunk 2]
```diff
@@ -2507,7 +2506,7 @@ def ExpandIntegrand(expr, x, extra=None):
                 if n/g == 2:
                     return s/(2*b*(r + s*u**g)) - s/(2*b*(r - s*u**g))
                 else:
-                    if CoprimeQ[m+g,n]:
```
Comment (parsoyaarihant) [inline sympy/integrals/rubi/utility_function.py]: add test for this case

File: sympy/integrals/rubi/utility_function.py [hunk 3]
```diff
@@ -11,23 +11,22 @@
...
-    factorint, Min, Max, sign, E, expand_trig, poly, apart, lcm, And, Pow, pi,
+from sympy import (Basic, exp, N, Wild, Sum, S, I, Mul,
+    Add, hyper, symbols, sqf_list, sqf, gcd, factorint,
+    Min, Max, sign, E, expand_trig, poly, apart, lcm, And, Pow, pi,
     zoo, oo, Integral)
 from mpmath import appellf1
-from sympy.functions.special.elliptic_integrals import elliptic_k, elliptic_f, elliptic_e, elliptic_pi
-from sympy.polys.polytools import poly_from_expr
+from sympy.functions.special.elliptic_integrals import elliptic_f, elliptic_e, elliptic_pi
+#from sympy.polys.polytools import poly_from_expr
```
Comment (parsoyaarihant) [inline sympy/integrals/rubi/utility_function.py]: remove these lines since it has no use

File: sympy/integrals/rubi/utility_function.py [hunk 4]
```diff
@@ -5224,7 +5223,7 @@ def FunctionOfDensePolynomialsQ(u, x):
         return True
     if PolynomialQ(u, x):
         return Length(Exponent(u,x,List))>1
-    return all(FunctionOfDensePolynomialsQ(i, v, x) for i in u.args)
+    return all(FunctionOfDensePolynomialsQ(i, x) for i in u.args)
```
Comment (parsoyaarihant) [inline sympy/integrals/rubi/utility_function.py]: add test for this case

File: sympy/integrals/rubi/utility_function.py [hunk 5]
```diff
@@ -6276,19 +6254,19 @@ def _FixSimplify():
     replacer.add(rule1)
 
     pattern2 = Pattern(UtilityOperator(Mul(WC('w', S(1)), Pow(u_, WC('m', S(1))), Pow(v_, n_))), CustomConstraint(lambda m: RationalQ(m)), CustomConstraint(lambda n: FractionQ(n)), CustomConstraint(lambda u: SqrtNumberSumQ(u)), CustomConstraint(lambda v: SqrtNumberSumQ(v)), CustomConstraint(lambda u: PositiveQ(u)), CustomConstraint(lambda v: PositiveQ(v)))
-    rule2 = ReplacementRule(pattern2, lambda n, m, u, v, w : With(List(Set(S('z'), Simplify(Mul(Pow(u, Mul(m, Pow(GCD(m, n), S(-1)))), Pow(v, Mul(n, Pow(GCD(m, n), S(-1)))))))), Condition(FixSimplify(Mul(w, Pow(S('z'), GCD(m, n)))), Or(AbsurdNumberQ(S('z')), SqrtNumberSumQ(S('z'))))))
+    rule2 = ReplacementRule(pattern2, lambda n, m, u, v, w : With(List(Set(S('z'), Simplify(Mul(Pow(u, Mul(m, Pow(GCD(m, n), S(-1)))), Pow(v, Mul(n, Pow(GCD(m, n), S(-1)))))))), Condition(FixSimplify(Mul(w, Pow(S('z'), GCD(m, n)))), Order(AbsurdNumberQ(S('z')), SqrtNumberSumQ(S('z'))))))
```
Comment (parsoyaarihant) [inline sympy/integrals/rubi/utility_function.py]: Why have you changed `Or` to `Order` ?

File: sympy/integrals/rubi/utility_function.py [hunk 6]
```diff
@@ -5339,6 +5338,7 @@ def EulerIntegrandQ(expr, x):
 
 def FunctionOfSquareRootOfQuadratic(u, *args):
     if len(args) == 1:
+        x = True
         x == args[0]
```
Comment (parsoyaarihant) [inline sympy/integrals/rubi/utility_function.py]: correct fix is to change this line to `x= args[0]`

**Comment by gxyd** [comment]
@alirizwi can you please revolve the merge conflicts here? Here's a guide to resolving conflicts https://help.github.com/articles/resolving-a-merge-conflict-using-the-command-line/

**Comment by jksuom** [comment]
https://github.com/sympy/sympy/issues/13283 was closed in https://github.com/sympy/sympy/pull/13329.

**Comment by gxyd** [comment]
Seems like the issue was indeed fixed by that PR. I am closing this pull request, @alirizwi please you can take up some other 'easy to fix' issue [here](https://github.com/sympy/sympy/issues?q=is%3Aopen+is%3Aissue+label%3A%22Easy+to+Fix%22).

**Pull request closed without merging**